### PR TITLE
2023-11 Changelog Entry

### PR DESCRIPTION
**Pull request by trevorgerhardt**
This references some changes made in https://github.com/conveyal/docs/pull/79, so that PR should be merged first.

### CODE REVIEW

File: changelog/2023-11-22.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+import Image from '../components/image'
+
+<Image
+  src='/img/changelog/202311/regional.png'
+  alt='Smoother regional analysis view'
+/>
+
+We have been hard at work on Conveyal's major release for 2023. This release includes **v7.0** of the R5 routing engine and...
```
Comment (abyrd) [inline changelog/2023-11-22.mdx]: Will this wording draw more attention to the fact that there are not many releases in 2023? Maybe we should say something like "Conveyal's late 2023 major release."

File: changelog/2023-11-22.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: 2023 Major Release
```
Comment (abyrd) [inline changelog/2023-11-22.mdx]: See comment below.

**Review by abyrd — COMMENTED** [review]
Wording of the changelog entries themselves looks good. Seeing the title in situ, I'm thinking we should de-emphasize the singular nature of the release. We could stick a more specific modifier in front of 2023 which would still be accurate, but not cause people to read and judge the feature list as harshly.

File: changelog/2023-11-22.mdx (hunk 1 above)
Comment (trevorgerhardt) [reply changelog/2023-11-22.mdx]: I agree, I suggest "November, 2023" as that would fit the style that we've used in the most recent changelog entries.

**Review by abyrd — APPROVED** [review]

**Pull request merged**